src/client/scripts/esm/chess/variants/variant.ts

```diff
@@ -382,12 +381,12 @@ function getGameRules(modifications: {
 	},
 	turnOrder?: string[],
 	promotionsAllowed?: {
-		white: string[],
-		black: string[]
+		white?: string[],
```

Comment:
Does it let us avoid redundancy here by just declaring the parameter as `modifications: GameRuleModifications`, instead of typing out the whole definition again?

Reply:
No, since there's an extra position property needed in the parameter.

Reply:
Hmm, we should probably make `position` a 2nd parameter now instead of merged with the GameRuleModifications. 🤔